src/ekomark/plots.py

```diff
@@ -209,7 +210,15 @@ def plot_operator(var_name, op, op_err, log_operator=True, abs_operator=True):
     return fig
 
 
-def save_operators_to_pdf(path, theory, ops, me: EKO, skip_pdfs, change_lab=False):
+def save_operators_to_pdf(
+    path,
+    theory,
+    ops,
+    me: EKO,
+    skip_pdfs,
+    rotate_to_evolution_basis: bool,
+    qed: bool = False,
```

Comment:
Docstrings are not updated anymore

Reply:
done in https://github.com/NNPDF/eko/pull/292/commits/4699064f9e4c5e368e148f1afb9614e0e9908689